frame/utility/src/lib.rs

```diff
@@ -497,6 +497,39 @@ pub mod pallet {
 			let res = call.dispatch_bypass_filter(frame_system::RawOrigin::Root.into());
 			res.map(|_| ()).map_err(|e| e.error)
 		}
+
+		/// Dispatches a function call with a provided origin. Unlike `dispatch_as`, this
+		/// function propagates errors returned by the wrapped `call`.
+		///
+		/// The dispatch origin for this call must be _Root_.
+		///
+		/// # <weight>
+		/// - O(1).
+		/// - Limited storage reads.
+		/// - One DB write (event).
+		/// - Weight of derivative `call` execution + T::WeightInfo::dispatch_as().
+		/// # </weight>
```

Comment:
```suggestion
		/// # <weight>
		/// - O(1).
		/// - Limited storage reads.
		/// - One DB write (event).
		/// - Weight of derivative `call` execution + T::WeightInfo::dispatch_as().
		/// # </weight>
```
We recently remove these (in https://github.com/paritytech/substrate/pull/13350) as the auto-generated weights are more expressive and correct.  
You can still keep the complexity notation if you want, but it should be pretty obvious anyway :man_shrugging:

Reply:
Ok, I rebased and updated to be inline with the `dispatch_as` doc string